api/lambdas/token-auth/index.js

```diff
@@ -50,6 +67,7 @@ exports.handler = async (event) => {
         };
       }
     } catch (error) {
+      // Don't we want to throw an error here?
       console.error(error);
     }
```

Comment:
Important question, I couldn't tell without digging deeper - I can review it if we don't remember easily.